frontend/src/metabase/dashboard/containers/DashboardHeader.jsx

```diff
@@ -285,16 +283,13 @@ class DashboardHeader extends Component {
     if (!isFullscreen && !isEditing && canEdit) {
       buttons.push(
         <Tooltip key="edit-dashboard" tooltip={t`Edit dashboard`}>
-          <a
-            data-metabase-event="Dashboard;Edit"
+          <DashboardHeaderButton
```

Comment:
Not sure why this is implemented using an anchor tag as it didn't have the `href` attribute.